python-saline.spec

```
@@ -20,6 +20,8 @@ BuildRequires:  python-rpm-macros
 BuildRequires:  systemd-rpm-macros
 BuildRequires:  %{python_module base}
 Requires:       %{python_module salt}
+Requires:       python3-CherryPy
+Requires:       python3-python-dateutil
```

In this case better to do it this way:
```suggestion
Requires:       python-CherryPy
Requires:       python-python-dateutil
Requires:       python-salt
```
What do you think?

---

Done!